src/Hosting.Services.Web/Middlewares/UserIdentity/IdentityProviders/EmailBasedUserIdentityProvider.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
+{
+	internal class EmailBasedUserIdentityProvider : IUserIdentityProvider
+	{
+        private const int MaxBytesPerChar = 4; // UTF-8
+		public int MaxBytesInIdentity { get; } = 256 * MaxBytesPerChar; // maximum email address length plus max bytes per char
+
+        private class UserEmail
+        {
+            public string Email { get; set; } = string.Empty;
+        }
+
+		public async Task<(bool success, int bytesWritten)> TryWriteBytesAsync(HttpContext context, Memory<byte> memory)
+		{
+			int bytesWritten = -1;
+            try{
```

Comment:
>{ [](http://example.com/codeflow?start=15&length=1)

Here and throughout this file, `{` should be on their own line following our coding conventions and standard C# practices

Reply:
fixed by `dotnet format`